### PR TITLE
Version 87

### PR DESCRIPTION
**Pull request by vinniefalco**
* Update appveyor for Boost branch
* Rename to BEAST_EXPECT
* variant fixes and tests
* Update README, add CONTRIBUTING.md and CODE_OF_CONDUCT.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+* Test developmental branches. When changes are made to Beast, they
+  are submitted as a
+  [Pull Request](https://github.com/boostorg/beast/pulls).
+  Everyone is welcome to examine the upcoming changes, test them,
+  and report on the results. This form of quality assurance is very
+  helpful so we can keep delivering a great product. Consider
+  subscribing to repository notifications to get alerted to new
+  pull requests.
+
+* Offer Code Review on
```
Comment (glenfe) [inline CONTRIBUTING.md]: Instead of "Offer Code Review on", just "Code review"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+* Provide
+  [feedback](https://github.com/boostorg/beast/issues).
+  about the library. If you have used, are using,
+  or are thinking about using the library we want to hear about it!
+  The more information you provide, the better we will be able
+  to ensure that Beast meets your needs. You can provide feedback
+  as an issue, or email the author directly.
+
+* Test developmental branches. When changes are made to Beast, they
```
Comment (glenfe) [inline CONTRIBUTING.md]: Why "developmental branches" here? i.e. Just "Test Beast" (presumably any branch, even master)

Comment (vinniefalco) [inline CONTRIBUTING.md]: willfix

File: CONTRIBUTING.md (hunk 1 above)
Comment (vinniefalco) [inline CONTRIBUTING.md]: willfix

**Pull request merged**